src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ClientTrackingConfiguration.kt

```diff
@@ -34,6 +34,8 @@ class ClientTrackingConfiguration(private val clientTrackingInterceptor: ClientT
         "/openapi/**",
         // HMPPS queue housekeeping endpoint
         "/queue-admin/retry-all-dlqs",
+        // temporary ETL endpoints
+        "/action-plans/review-schedules/etl/**",
```

Comment:
Exclude this URL from the client tracking bean (that would otherwise try to process the bearer token ... which our new endpoint does not carry)